dips/dip-20.md

```diff
@@ -171,79 +168,60 @@ determining exchange rates for value transfer between different currencies.
 
 ### `is_synthetic`
 
-This defines whether the given currency is synthetic or an SCS. In the case of
+This immutable field defines whether the given currency is synthetic or an SCS. In the case of
 [`Diem::register_SCS_currency`](https://github.com/diem/diem/blob/master/language/stdlib/modules/doc/Diem.md#function-register_scs_currency)
 the function sets this to `false`. For
 [`Diem::register_currency`](https://github.com/diem/diem/blob/master/language/stdlib/modules/doc/Diem.md#function-register_currency)
-this must be provided to the function.
-
-#### Mutability
-Immutable
+this value must be provided to the function.
 
 ### <a name="scaling_factor">`scaling_factor`</a>
 
-This defines what fraction each unit of value on-chain represents in the
+This immutable field defines what fraction each unit of value on chain represents in the
 currency. As an example, the on-chain unit of value for XUS is
 one-millionth of an XUS, therefore XUS's `scaling_factor` is 1,000,000.
 
-#### Mutability
-Immutable
-
 ### `fractional_part`
 
-This defines the truncation that needs to be performed when transferring
-from the on-chain representation to the smallest denomination real-world
-coin for that currency off-chain. For example, for XUS the smallest
-off-chain denomination coin is one cent (1/100th of an XUS), so the
-`fractional_part` for XUS would be 100. If you wanted to translate from an
-on-chain value `v` to a number in XUS's currency units, you would multiply
-by 1,000,000 (XUS's `scaling_factor`) and then truncate all digits after the
-hundredths place.
-
-#### Mutability
-Immutable
+This immutable field defines the truncation that needs to be performed when transferring
```

Comment:
The phrasing of this needs some work. In many situations, it may be better to round instead of truncating, so we should be agnostic here. We should also avoid saying "needs to be performed" -- some applications may prefer to use smaller fractional units.